src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    public static final double PI = 3.14159265358979323846;
```

Comment:
Consider using `Math.PI` instead of a hardcoded PI value. This ensures better precision and maintainability as `Math.PI` is a predefined constant in Java.